### ISSUE TITLE
slack notification when pipeline job finishes

### ISSUE THREAD

**Issue by xavierchow** [Wiredcraft/pipelines#105]
Currently the pipeline is working as an asynchrous pattern, i.e. I trigger the build and wait for a few minutes to let it finishes.
As some of the pipeline job could be quite slow it lasts for 5~10 min and I won't sit on the pipeline page to wait for it, I sometimes forget I trigger a build.
It's would be helpful if we can get a slack notification which pings who triggers the build when it's done.

**Comment by hunvreus** [comment]
A better integration with Slack AND GitHub would be good TBH.

- For GitHub, having the ability of leaving messages for builds (like TravisCI).
- For Slack, receiving status but also triggering from Slack channel.

**Comment by xuqingfeng** [comment]
This issue was addressed in https://github.com/Wiredcraft/pipelines/pull/106 .

(A new input field need to be added in the ui though)

@xavierchow Let me know which project you want to have it, I can update accordingly.

**Comment by xavierchow** [comment]
omnisaas staging pipeline if it's possible

**Comment by kaleocheng** [comment]
closing as done

**Comment by xavierchow** [comment]
@kaleocheng 
> a slack notification which pings who triggers the build when it's done

Is there a way to `@` the one who triggers the pipeline? or at least include the `trigger info` in the slack notification?

**Comment by kaleocheng** [comment]
@xavierchow yes, we support something like this, I can setup it for you later
![图片](https://user-images.githubusercontent.com/7939352/168512463-740fcac6-3e6c-43e0-a6f3-de3a32b55cc6.png)

( I may misunderstand your request in here,   we still need a better integration for this issue, reopening  )

**Comment by xavierchow** [comment]
thanks, is there any concern that prevents making the `slack_mention` enabled as the default?
I don't see this input box from blue pipeline
<img width="993" alt="image" src="https://user-images.githubusercontent.com/2748884/168519499-94180afe-58ae-4ed8-8a72-d8f564cb41cf.png">
